.gitignore

```diff
@@ -47,6 +47,8 @@ storybook-static/
 node_modules/
 jspm_packages/
 
+!packages/cli/src/__mocks__/**/node_modules
+
```

Comment:
**question (non-blocking):** why we need to add node_modules from cli?

Reply:
I think this is related to the faststore platform initiative. Not introduced in this PR.
Would you happen to know @gvc, @vaporwavie?

Reply:
This was implemented by @gvc but I support what you mentioned @eduardoformiga. It was introduced on this commit that refers adding `core` for monorepos: https://github.com/vtex/faststore/commit/e6e3ab7710ce2418afb07127ceab3c1bb8e2bcfc.